tests/Microsoft.Identity.Test.Integration.netfx/HeadlessTests/OboTests.cs

```diff
@@ -480,6 +480,56 @@ public async Task AcquireTokenByObo_NormalOboThenLongRunningInitiate_WithTheSame
             Assert.AreEqual(1, cca.UserTokenCacheInternal.Accessor.GetAllRefreshTokens().Count);
         }
 
+        [TestMethod]
+        public async Task AcquireTokenByObo_LongRunning_WithDifferentScopes_TestAsync()
```

Comment:
Wondering if we should add another test method, similar to this one but when there's no AT in the cache at all, since we have that slight branching when we check if GetAllAT returns 0 tokens? Maybe a unit test instead of integration?